packages/cxl-ui/scss/global/cxl-institute-layout.scss

```diff
@@ -104,3 +104,74 @@ cxl-institute-layout {
     }
   }
 }
+cxl-institute-layout[theme~="blog-post"] {
+  > .entry {
+    .entry-header {
+      margin-bottom: var(
+        --lumo-space-xl
+      ) !important; /* stylelint-disable-line declaration-no-important */
+
+      label {
+        font-size: var(--lumo-font-size-l);
+        color: var(--lumo-primary-color);
+      }
+
+      .avatar {
+        border-radius: 50%;
+      }
+
+      .author-image {
+        float: left;
+        margin-right: var(--cxl-wrap-padding);
+      }
+
+      .blog-byline {
+        font-size: var(--lumo-font-size-xl);
+      }
+
+      .entry-author {
+        padding: var(--lumo-space-l) 0;
+        border-bottom: 1px solid var(--lumo-contrast-50pct);
+
+        /* stylelint-disable-next-line selector-max-compound-selectors */
+        span {
+          display: block;
+        }
+      }
+    }
+
+    > * {
+      font-size: var(--lumo-font-size-l);
+    }
+  }
+
+  > .widget {
+    padding: var(--cxl-wrap-padding);
+
+    &#widget-webinar {
+      color: var(--lumo-base-color);
+      background-color: var(--lumo-contrast);
+    }
+
+    &#widget-video-course {
+      background-color: var(--lumo-shade-10pct);
```

Comment:
I'm not sure we will hardcoding specific widget styles into the layout (too many to keep track of). These can be inline "app" styles in Storybook and we'll figure out how it'll work in WP.